### ISSUE TITLE
infiniband not detected by rockstor UI

### ISSUE THREAD

**Issue by ClonedPuppy** [rockstor/rockstor-core#1641]
Hi, just a small request to please add support or just plainly ignore the following error msg:

ERROR [storageadmin.views.network:85] Unknown ctype: infiniband config: {}

It's clogging up the logs...

Thanks!

**Comment by phillxnet** [comment]
@Bazed Thanks for creating the issue. I'm linking back to your associated forum thread:
https://forum.rockstor.com/t/infiniband-rockstor/2785

Also observed with default DHCP install on a KVM VMM VM:
```
[04/Feb/2017 15:47:06] ERROR [storageadmin.views.network:81] Unknown ctype: bridge config: {}
```
![network-log-issue](https://cloud.githubusercontent.com/assets/2521585/22619732/219addcc-eaf3-11e6-89e8-12e0e8cdb419.jpg)

**Comment by MFlyer** [comment]
debug over system/network.py for a docker connection

```
root@rockstone:views# nmcli c show 8f7a9186-884b-4bc7-8c01-c5f00d35d35a
connection.id:                          docker0
connection.uuid:                        8f7a9186-884b-4bc7-8c01-c5f00d35d35a
connection.stable-id:                   --
connection.interface-name:              docker0
connection.type:                        bridge
connection.autoconnect:                 no
connection.autoconnect-priority:        0
connection.timestamp:                   1486224956
connection.read-only:                   no
connection.permissions:
connection.zone:                        --
connection.master:                      --
connection.slave-type:                  --
connection.autoconnect-slaves:          -1 (default)
connection.secondaries:
connection.gateway-ping-timeout:        0
connection.metered:                     unknown
connection.lldp:                        -1 (default)
ipv4.method:                            manual
ipv4.dns:
ipv4.dns-search:
ipv4.dns-options:                       (default)
ipv4.dns-priority:                      100
ipv4.addresses:                         172.17.0.1/16
ipv4.gateway:                           --
ipv4.routes:
ipv4.route-metric:                      -1
ipv4.ignore-auto-routes:                no
ipv4.ignore-auto-dns:                   no
ipv4.dhcp-client-id:                    --
ipv4.dhcp-timeout:                      0
ipv4.dhcp-send-hostname:                yes
ipv4.dhcp-hostname:                     --
ipv4.dhcp-fqdn:                         --
ipv4.never-default:                     no
ipv4.may-fail:                          yes
ipv4.dad-timeout:                       -1 (default)
ipv6.method:                            ignore
ipv6.dns:
ipv6.dns-search:
ipv6.dns-options:                       (default)
ipv6.dns-priority:                      100
ipv6.addresses:
ipv6.gateway:                           --
ipv6.routes:
ipv6.route-metric:                      -1
ipv6.ignore-auto-routes:                no
ipv6.ignore-auto-dns:                   no
ipv6.never-default:                     no
ipv6.may-fail:                          yes
ipv6.ip6-privacy:                       -1 (unknown)
ipv6.addr-gen-mode:                     stable-privacy
ipv6.dhcp-send-hostname:                yes
ipv6.dhcp-hostname:                     --
ipv6.token:                             --
bridge.mac-address:                     --
bridge.stp:                             no
bridge.priority:                        32768
bridge.forward-delay:                   15
bridge.hello-time:                      2
bridge.max-age:                         20
bridge.ageing-time:                     300
bridge.multicast-snooping:              yes
GENERAL.NAME:                           docker0
GENERAL.UUID:                           8f7a9186-884b-4bc7-8c01-c5f00d35d35a
GENERAL.DEVICES:                        docker0
GENERAL.STATE:                          activated
GENERAL.DEFAULT:                        no
GENERAL.DEFAULT6:                       no
GENERAL.VPN:                            no
GENERAL.ZONE:                           --
GENERAL.DBUS-PATH:                      /org/freedesktop/NetworkManager/ActiveConnection/1
GENERAL.CON-PATH:                       /org/freedesktop/NetworkManager/Settings/1
GENERAL.SPEC-OBJECT:                    /
GENERAL.MASTER-PATH:                    --
IP4.ADDRESS[1]:                         172.17.0.1/16
IP4.GATEWAY:
IP6.GATEWAY:
```

Well, I can't define it a bug at all, we just don't handle it (only eth, bond and team, no bridge, wifi, wimax, infiniband and others), but on the other side having edit over docker0 I get a bond with round robin balance O_o??

**Comment by FroggyFlox** [comment]
I just noticed this one... I agree that this "error" in the logs isn't really one but simply represents that we do not support anything that isn't Ethernet, team, or bond for now.
In my current work on docker networking (#2003, #2009), I currently removed this logging as error after adding the `bridge` ctype, but I can keep it in if I improve the wording.
Not a fix for this issue but thought I would mention this here for reference as the corresponding code will change slightly upon completion of the two issues referenced above.

**Comment by FroggyFlox** [comment]
As mentioned above, it is my understanding that this "log spamming" has been resolved during #2207  (see https://github.com/rockstor/rockstor-core/pull/2207#issuecomment-678850905). I'm thus closing this issue as a result.
